packaging/ansible-runner-service-project/project/roles/gluster-replace-peers/tasks/main.yml

```diff
@@ -36,28 +36,27 @@
 
 # Detach the old host, to replace host with different FQDN usecase
 - name: Detach the peer, in the case of different host replacement
-  command: "gluster peer detach {{ gluster_maintenance_old_node }} force"
+  ansible.builtin.command: "gluster peer detach {{ gluster_maintenance_old_node }} force"
   when: gluster_maintenance_old_node != gluster_maintenance_new_node
 
 - name: Force removal of old node peer in new node
-  file:
+  ansible.builtin.file:
     path: "{{ glusterd_libdir }}/peers/{{ old_node_uuid.stdout | trim }}"
     state: absent
   when: gluster_maintenance_old_node != gluster_maintenance_new_node
-  delegate_to : "{{ gluster_maintenance_new_node }}"
+  delegate_to: "{{ gluster_maintenance_new_node }}"
   connection: ssh
 
 - name: Restart glusterd on the new node
-  connection: ssh
```

Comment:
Why was it here (and in the other places below) and now we remove it?  Better to do this in a commit separate from the purely formal changes.